src/components/Personalization/index.js

```diff
@@ -44,35 +49,51 @@ const createCollect = eventManager => {
   };
 };
 
+const validateEvent = (event, logger) => {
+  const warnings = event.validate();
```

Comment:
I had some thoughts on `event.validate()`. Let's talk about it in our next Alloy standup.